### PR TITLE
Update CNAB-simple example to reflect the UX changes

### PR DESCRIPTION
**Pull request by carolinebriaud**
Signed-off-by: Caroline Briaud <caroline.briaud@docker.com>

**- What I did**
Update CNAB-simple example to reflect the UX changes.
Add final step (remove running app) to hello-world example.

**- How to verify it**
Check the README file

**- A picture of a cute animal (not mandatory)**
34053984270_99b1cb12c2_b

### CODE REVIEW

File: examples/cnab-simple/README.md [hunk 1]
```diff
@@ -1,60 +1,127 @@
...
+RUNNING APP       APP NAME        LAST ACTION  RESULT   CREATED         MODIFIED          REFERENCE
+mycnabexample     hello (0.2.0)   install      success  15 minutes ago  15 minutes ago    docker.io/myrepo/cnab-example:1.0.0
 ```
+
+Inspect your running app using the `docker app inspect`command.
+
+```shell
+$ docker app inspect mycnabexample --pretty
+Running App:
+  Name: titi
```
Comment (silvin-lubecki) [inline examples/cnab-simple/README.md]: titi? 🤣

File: examples/cnab-simple/README.md [hunk 2]
```diff
@@ -1,60 +1,127 @@
-# Docker Application to CNAB
+# Example: From Docker App to CNAB
```
Comment (silvin-lubecki) [inline examples/cnab-simple/README.md]: I wonder if all this example should be merged with the hello world example, as it does not show more things than the hello-world example already has, in its `.dockerapp` structure 🤔

File: examples/cnab-simple/README.md [hunk 3]
```diff
@@ -1,60 +1,119 @@
...
+Now we are going to build an App image from this App definition.
+
+```shell
+$ docker app build . -f hello.dockerapp -t myrepo/cnab-example:1.0.0
+[+] Building 0.6s (6/6) FINISHED
+(...) (Build output)
+sha256:ee61121d6bff0266404cc0077599c1ef7130289fec721
+```                                                                              
+
+*Note that a `bundle.json` file has been created in the `/Users/username/.docker/app/bundles/docker.io/myrepo/cnab-example/_tags/1.0.0` directory.*
```
[the remark this answers is not in the document]
Comment (carolinebriaud) [reply examples/cnab-simple/README.md]: ok, done.

File: examples/cnab-simple/README.md [hunk 4]
```diff
@@ -1,60 +1,119 @@
...
-Uninstall the application installation:
+### Run the app by passing a `bundle.json` file
+
+To demonstrate that Docker App is an implementation of [CNAB](https://cnab.io/), it is also possible to directly run the `bundle.json` file (or any other CNAB bundle) using the `--cnab-bundle-json` experimental flag. 
+
+*Note: To use this flag, you have to enable the experimental mode for the Docker CLI first.*
 
-```console
-$ docker app uninstall hello
+Open the `/Users/username/.docker/config.json` file in a text editor and change the `"experimental"` field to `"enabled"`.
```
[the remark this answers is not in the document]
Comment (carolinebriaud) [reply examples/cnab-simple/README.md]: Done

File: examples/cnab-simple/README.md (hunk 1 above)
Comment (carolinebriaud) [reply examples/cnab-simple/README.md]: why should we always use toto? 🤣

File: examples/cnab-simple/README.md (hunk 2 above)
Comment (carolinebriaud) [reply examples/cnab-simple/README.md]: Yes I agree. I would not remove it. But we might want to use another simple App definition. Any proposal?

Comment (carolinebriaud) [reply examples/cnab-simple/README.md]: We could maybe merge it as it is, then think about proposing another app definition in another step?

Comment (silvin-lubecki) [reply examples/cnab-simple/README.md]: Ok for a follow-up 👍

**Pull request merged**